examples/30_extended/task_manual_iteration_tutorial.py

```diff
@@ -0,0 +1,181 @@
+# defined by the task:
+
+n_repeats, n_folds, n_samples = task.get_split_dimensions()
+
+####################################################################################################
+# * ``n_repeats``: Number of times the model quality estimation is performed
+# * ``n_folds``: Number of folds per repeat
+# * ``n_samples``: How many data points to use. This is only relevant for learning curve tasks
+#
+# A list of all available estimation procedures is available
+# `here <https://www.openml.org/search?q=%2520measure_type%3Aestimation_procedure&type=measure>`_.
+#
+# Task ``233`` is a simple task using the holdout estimation procedure and therefore has only a
+# single repeat, a single fold and a single sample size:
+
+print(
+    'Task {}: number of repeats: {}, number of folds: {}, number of samples {}.'.format(
+        task_id, n_repeats, n_folds, n_samples,
+    )
+)
+
+####################################################################################################
+# We can now retrieve the train/test split for this combination of repeats, folds and number of
+# samples (indexing is zero-based). Usually, one would loop over all repeats, folds and sample
+# sizes, but we can neglect this here as there is only a single repetition.
+
+train_indices, test_indices = task.get_train_test_split_indices(
+    repeat=0,
+    fold=0,
+    sample=0,
+)
+
+print(train_indices.shape, train_indices.dtype)
+print(test_indices.shape, test_indices.dtype)
+
+####################################################################################################
+# And then split the data based on this:
+
+X, y, _, _ = task.get_dataset().get_data(task.target_name)
+X_train = X.loc[train_indices]
+y_train = y[train_indices]
+X_test = X.loc[test_indices]
+y_test = y[test_indices]
+
+print(
+    'X_train.shape: {}, y_train.shape: {}, X_test.shape: {}, y_test.shape: {}'.format(
+        X_train.shape, y_train.shape, X_test.shape, y_test.shape,
+    )
+)
+
+####################################################################################################
+# Obviously, we can also retrieve cross-validation versions of the dataset used in task ``233``:
+
+task_id = 3
+task = openml.tasks.get_task(task_id)
+n_repeats, n_folds, n_samples = task.get_split_dimensions()
+print(
+    'Task {}: number of repeats: {}, number of folds: {}, number of samples {}.'.format(
+        task_id, n_repeats, n_folds, n_samples,
+    )
+)
+
+####################################################################################################
+# And then perform the aforementioned iteration over all splits:
+for repeat_idx in range(n_repeats):
+    for fold_idx in range(n_folds):
+        for sample_idx in range(n_samples):
+            train_indices, test_indices = task.get_train_test_split_indices(
+                repeat=0,
+                fold=0,
+                sample=0,
+            )
```

Comment:
Should it not be ` repeat = repeat_idx, fold = fold_idx, sample=sample_idx`

Reply:
Yes, will fix this and the other, similar comments.